dev_guide/idling_unidling_services.adoc

```diff
@@ -0,0 +1,63 @@
+[[dev-guide-idling-unidling-services]]
+= Idling and Unidling Services
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+You can idle services in order to reduce the consumption of resources.
+
+If a set of services (or an application) are taking up resources, but still
+might need to be used in that state at a later time, you can idle the services
+until they need to be used again.
+
+[[idling-unidling-idling-services]]
+== Idling Services
+
+Idling a service finds the scalable resources (deployment configurations,
+replication controllers, and others) associated with a service. Idling a service
+marks it as idled by scaling down the resources to zero replicas.
+
+You can use the `oc idle` command to
+xref:../dev_guide/idling_unidlge_pods.adoc#idling-unidling-idling-single-services[idle
+a single service], or use the `--resource-names-file` option to
+xref:../dev_guide/idling_unidlge_pods.adoc#idling-unidling-idling-multiple-services[idle
+multiple services].
+
+[[idling-unidling-idling-single-services]]
+=== Idling Single Services
+
+Idle a single service with the following command:
+
+----
+$ oc idle <service>
+----
+
+[[idling-unidling-idling-multiple-services]]
+=== Idling Multiple Services
+
+Idle multiple services by creating a list of the desired services, then using the `--resource-names-file` option with the `oc idle` command.
+
+This is helpful if an application spans across a set of services. Idling them all at one time would be the most efficient way.
```

Comment:
While this is true, it's more of "this is useful for bulk-idling applications as part of a script"